include/padmgr.h

```diff
@@ -70,4 +70,37 @@ void PadMgr_RumbleReset(PadMgr* padMgr);
 void PadMgr_RumbleSetSingle(PadMgr* padMgr, u32 port, u32 rumble);
 void PadMgr_RumbleSet(PadMgr* padMgr, u8* enable);
 
+// Retrace callback
+
+/**
+ * Sets the padmgr retrace callback that runs while waiting for controller input. The callback may be passed a single
+ * user-provided argument. The callback function should be `void (*)(PadMgr*, void*)`.
+ *
+ * @param callback callback to run before rumble state is updated for the current VI
+ * @param arg the argument to pass to the calback
+ * 
+ * @see PadMgr_UnsetRetraceCallback
+ */
+#define PadMgr_SetRetraceCallback(padmgr, callback, arg) \
```

Comment:
This macro shouldn't be named like a function, since `padmgr` is evaluated twice by the expansion
(in general naming macros like functions is risky)